pull_data.php

```diff
@@ -16,37 +14,55 @@ function about_output(pdo $db) :array
 $about_result=about_output($db);
 
 
+
 /*
- * takes the array $about_result and turns it into a single key value of bio
+ * takes the array $about_result and turns it into a single key value of bio and finds out if the value is null
  *
  * @$param array this is an array of fields from the about_me section of the db
  *
- * @returns string values from the key bio
+ * @returns string values from the key bio or if value is null it returns no
```

Comment:
You either return the Bio as a string or a Boolean for which your return hints are not giving.